cw_decred/lib/api/libdcrwallet.dart

```diff
@@ -229,3 +229,28 @@ String signMessage(
   );
   return res.payload;
 }
+
+String? newExternalAddress(String walletName) {
+  final cName = walletName.toCString();
+  final res = executePayloadFn(
+    fn: () => dcrwalletApi.newExternalAddress(cName),
+    ptrsToFree: [cName],
+    skipErrorCheck: true, // errCode is checked below, before checking err
+  );
+  if (res.errCode == ErrCodeNotSynced) {
+    // Wallet is not synced. We do not want to give out a used address so give
+    // nothing.
+    return null;
+  }
```

Comment:
Why not just throw this error as well?

Reply:
I believe this one happens as you start the wallet, before you ever go to the receive screen. So it would error on every new wallet which wouldn't look good. Want to do something more obvious.